<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowFilms</class>
 <widget class="QWidget" name="ShowFilms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>230</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1004</width>
        <height>620</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item alignment="Qt::AlignLeft">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="pic.qrc">
                <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>100</horstretch>
                <verstretch>200</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>236</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>236</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Мои Билеты</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>600</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit">
              <property name="font">
               <font>
                <italic>false</italic>
                <underline>false</underline>
               </font>
              </property>
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhTime</set>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="currentSection">
               <enum>QDateTimeEdit::HourSection</enum>
              </property>
              <property name="displayFormat">
               <string>H:mm</string>
              </property>
              <property name="calendarPopup">
               <bool>false</bool>
              </property>
              <property name="timeSpec">
               <enum>Qt::LocalTime</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>551</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
